udpate README.md file (#2388)
ColoredCow/portal: README.md

### Installation

Use our [installation guidelines](./docs/installation.md) to set up the project on your local.

> For the database UAT link, you can use the following one (https://drive.google.com/file/d/1LQ2Cqd9dbY8G1WqsbfmF2D5h8rMarKjm/view?usp=sharing)

Reviewer: ```suggestion
To download the UAT database, use the following [link]((https://drive.google.com/file/d/1LQ2Cqd9dbY8G1WqsbfmF2D5h8rMarKjm/view?usp=sharing)) 
```
